composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "networkrailbusinesssystems/activity-log",
-    "description": "Ahhhhh",
+    "description": "A library that contains the common activity log components supporting our GovUk systems",
```

Comment:
> A library that contains the common activity log components supporting our GovUk systems

We already know that this is a library, so we can shorten it a little. GOV.UK is the correct way to refer to it.

> Common activity log components for GOV.UK systems

Sounds fine to me.